'react-redbox' is not in the npm registry

I double checked on several servers:
```
npm ERR! node v5.7.0
npm ERR! npm  v3.6.0
npm ERR! code E404

npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/react-redbox
npm ERR! 404
npm ERR! 404  'react-redbox' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! Please include the following file with any support request:
npm ERR!     /tmp/npm-debug.log
```

Oops, that's my bad.  It's redbox-react ☺️  will fix the README when i get home. 

On 18 March 2016 4:50:24 PM SAST, Yihong <notifications@github.com> wrote:
>I double checked on several servers:
>```
>npm ERR! node v5.7.0
>npm ERR! npm  v3.6.0
>npm ERR! code E404
>
>npm ERR! 404 Registry returned 404 for GET on
>https://registry.npmjs.org/react-redbox
>npm ERR! 404
>npm ERR! 404  'react-redbox' is not in the npm registry.
>npm ERR! 404 You should bug the author to publish it (or use the name
>yourself!)
>npm ERR! 404
>npm ERR! 404 Note that you can also install from a
>npm ERR! 404 tarball, folder, http url, or git url.
>
>npm ERR! Please include the following file with any support request:
>npm ERR!     /tmp/npm-debug.log
>```
>
>---
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/gadicc/meteor-react-hotloader/issues/11

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

--- a/README.md
+++ b/README.md
@@ -40,7 +40,7 @@ react error catching.
 
 *Use with correct Meteor release, currently 1.3-rc.2*
 
-1. In your project root, `npm install --save react-transform-hmr react-transform-catch-errors react-redbox`.
+1. In your project root, `npm install --save react-transform-hmr react-transform-catch-errors redbox-react`.
 1. Make sure you have a `.babelrc` in your project root that resembles the
 sample at the end of this README.
 1. Edit your `.meteor/packages` and replace `ecmascript` with `gadicc:ecmascript-hot@0.0.11-rc.2`

